client/router.coffee

```diff
@@ -106,6 +110,10 @@ Router.route "/user-event/:_id/:_view?",
     articles: Articles.find({'userEventId': @params._id}, {sort: {publishDate: -1}}).fetch()
     incidents: Incidents.find({'userEventId': @params._id}, {sort: {date: -1}}).fetch()
 
+Router.route "/smart-event/:_id/:_view?",
```

Comment:
If we're going to use tabs on the smart event page I think we should keep it, otherwise let's take it out.